### DISCUSSION TITLE
Ayuda con mensajes con links creados desde bots

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#97] [2. Questions and Answers] [answered]
Buenas a todos,

Estoy intentando crear un mensaje text/system que contenga un link pero que no tenga el redirect para transparentar al usuario hacia dónde va a ir antes de hacer click y principalmente para evitar que recargue toda la pagina para links que están dentro de Cotalker (como al pasar de un canal a otro). Antes la función de redirect no se aplicaba a los mensajes enviados desde bots pero ahora sí. En mi opinión, si se puede desactivar la creación de links redirect para las compañías individualmente o dejarlo como antes, que no afecte a los mensajes enviados por bots, sería mejor.

Intenté enviar el link utilizando markdown pero no resulta ya que la sintaxis de markdown utiliza paréntesis
"$JOIN##Access in (bracket)this link(/bracket)(https://web.cotalker.com/channel/#($OUTPUT#create_task#task|channel)#)"

Toda ayuda se agradece.

Saludos.

**Comment by NicolasPLG** [comment]
@SeIturra creo que te falta para el # (O sea #(Access y lo demás), podrías probar eso

**Reply by seiturra** [reply]
@NicolasPLG para textos que no son dinámicos (como $VALUE...) no es necesario poner paréntesis

**Comment by seiturra** [comment]
Usando 
"$JOIN##Access here: https://web.cotalker.com/channel/#($OUTPUT#create_task#task|channel)", 
Ahora no crea el redirect. Creo que cambié el uso de paréntesis. Con esto se soluciona, supongo que nunca hubo problema y estaba creando el link de una manera distinta a la habitual sin darme cuenta. 

Gracias.

**Comment by acedward** [comment, marked as answer]
Ahora hay que escapar literales con el nuevo motor de COTLang
```
$JOIN##```Access in [this link](https://web.cotalker.com/channel/```#($OUTPUT#create_task#task|channel)#```)```
```

**Reply by acedward** [reply]
los triple backtick ` permiten escapar literales
